js/skin.js

```diff
@@ -128,8 +128,8 @@ var Skin = createReactClass({
         isVrMouseMove: true
       });
       if (typeof this.props.controller.onTouchMove === 'function') {
-        var coords = Utils.getCoords(event);
-        var params = this.getDirectionParams(coords.x, coords.y);
+        let coords = Utils.getCoords(event);
+        let params = this.getDirectionParams(coords.x, coords.y);
```

Comment:
coords and params are const inside this block as well

Reply:
\+